<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIClass</class>
 <widget class="QMainWindow" name="GUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QTabWidget" name="main_tab_widget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="senior_tab">
       <attribute name="title">
        <string>Senior clearance</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="senior_table">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <column/>
          <column/>
          <column/>
          <column/>
          <column/>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="iDLabel">
              <property name="text">
               <string>&amp;ID:</string>
              </property>
              <property name="buddy">
               <cstring>ID_field</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="ID_field">
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="measurementLabel">
              <property name="text">
               <string>&amp;Measurement:</string>
              </property>
              <property name="buddy">
               <cstring>measurement_field</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="measurement_field">
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="clarityLabel">
              <property name="text">
               <string>&amp;Clarity:</string>
              </property>
              <property name="buddy">
               <cstring>clarity_field</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="clarity_field">
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="quantityLabel">
              <property name="text">
               <string>&amp;Quantity:</string>
              </property>
              <property name="buddy">
               <cstring>quantity_field</cstring>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="quantity_field">
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="photographLabel">
              <property name="text">
               <string>&amp;Photograph:</string>
              </property>
              <property name="buddy">
               <cstring>photograph_field</cstring>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="photograph_field">
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QPushButton" name="senior_add_evidence_button">
              <property name="text">
               <string>Add</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="senior_update_evidence_button">
              <property name="text">
               <string>Update</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="senior_delete_evidence_button">
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="standard_tab">
       <attribute name="title">
        <string>Standard clearance</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTableWidget" name="standard_table">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="columnCount">
             <number>5</number>
            </property>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="standard_save_button">
            <property name="text">
             <string>&gt;</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="standard_user_table">
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="columnCount">
             <number>5</number>
            </property>
            <column/>
            <column/>
            <column/>
            <column/>
            <column/>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0">
           <layout class="QFormLayout" name="formLayout_2">
            <property name="formAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="measurementLabel_2">
              <property name="text">
               <string>Measurement:</string>
              </property>
              <property name="buddy">
               <cstring>standard_measurement_filter_field</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="standard_measurement_filter_field">
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="minQuantityLabel">
              <property name="text">
               <string>Min. Quantity:</string>
              </property>
              <property name="buddy">
               <cstring>standard_quantity_filter_field</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="standard_quantity_filter_field">
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="resultLabel">
              <property name="text">
               <string>Result:</string>
              </property>
              <property name="buddy">
               <cstring>standard_filter_result_field</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="standard_filter_result_field">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QPushButton" name="open_button">
              <property name="text">
               <string>Open (file) in another program</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="open_locally_button">
              <property name="text">
               <string>Open in new window</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="standard_filter_button">
              <property name="text">
               <string>Filter</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="standard_filter_button_next">
              <property name="text">
               <string>Next</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="chart_tab">
       <attribute name="title">
        <string>Chart</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="chart_tab_layout"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="undo_button">
        <property name="text">
         <string>Undo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="redo_button">
        <property name="text">
         <string>Redo</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>main_tab_widget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>GUIClass</receiver>
   <slot>tabwidget_tab_changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>undo_button</sender>
   <signal>clicked()</signal>
   <receiver>GUIClass</receiver>
   <slot>undo_button_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>624</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>643</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>redo_button</sender>
   <signal>clicked()</signal>
   <receiver>GUIClass</receiver>
   <slot>redo_button_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>626</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>642</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>senior_add_evidence_button</sender>
   <signal>clicked()</signal>
   <receiver>GUIClass</receiver>
   <slot>senior_add_button_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>604</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>678</x>
     <y>635</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>senior_update_evidence_button</sender>
   <signal>clicked()</signal>
   <receiver>GUIClass</receiver>
   <slot>senior_update_button_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>633</x>
     <y>511</y>
    </hint>
    <hint type="destinationlabel">
     <x>677</x>
     <y>621</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>senior_delete_evidence_button</sender>
   <signal>clicked()</signal>
   <receiver>GUIClass</receiver>
   <slot>senior_delete_button_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>615</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>675</x>
     <y>606</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>senior_table</sender>
   <signal>cellDoubleClicked(int,int)</signal>
   <receiver>GUIClass</receiver>
   <slot>senior_table_cell_doubleclick(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>588</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>standard_save_button</sender>
   <signal>clicked()</signal>
   <receiver>GUIClass</receiver>
   <slot>standard_save_button_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>standard_filter_button</sender>
   <signal>clicked()</signal>
   <receiver>GUIClass</receiver>
   <slot>standard_filter_button_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>683</x>
     <y>523</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>standard_filter_button_next</sender>
   <signal>clicked()</signal>
   <receiver>GUIClass</receiver>
   <slot>standard_filter_button_next_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>679</x>
     <y>566</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_button</sender>
   <signal>clicked()</signal>
   <receiver>GUIClass</receiver>
   <slot>standard_open_button_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>656</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>682</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open_locally_button</sender>
   <signal>clicked()</signal>
   <receiver>GUIClass</receiver>
   <slot>standard_open_locally_button_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>500</y>
    </hint>
    <hint type="destinationlabel">
     <x>680</x>
     <y>470</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tabwidget_tab_changed(int)</slot>
  <slot>undo_button_click()</slot>
  <slot>redo_button_click()</slot>
  <slot>senior_add_button_click()</slot>
  <slot>senior_update_button_click()</slot>
  <slot>senior_delete_button_click()</slot>
  <slot>senior_table_cell_doubleclick(int,int)</slot>
  <slot>standard_save_button_click()</slot>
  <slot>standard_filter_button_click()</slot>
  <slot>standard_filter_button_next_click()</slot>
  <slot>standard_open_button_click()</slot>
  <slot>standard_open_locally_button_click()</slot>
 </slots>
</ui>
